scraps.rb

```diff
@@ -0,0 +1,51 @@
+This is a scraps file.  Does this work ok?
```

Comment:
Not sure what this file does, but if it just a file for notes, might be best to add to the `.gitignore` file so it doesn't get added for code reviews and merges.